src/ddsrt/cmake/thread_list.c

```diff
@@ -0,0 +1,18 @@
+/*
```

Comment:
This is not going to work(?) as it's not in the list of files that CMake checks. You can add a `threads.c` and define `cmake_HAVE_THREADS` to always be one and `cmake_HAVE_THREAD_LIST` either `true` or `false` based on the header.

Reply:
Hmm ... I'll delete the test. The ``DDSRT_HAVE_THREAD_LIST`` macro is what matters for now, and whether or not it is true has no bearing on which files need to be taken into account. So there is no need to get CMake involved for now.